components/finalMessage/finalMessage.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+
+import { Typography, Button } from '@mui/material'
+
+import { StyledFinalMessageContent, StyledTextArea } from '../../pageStyles/emailSender.styles'
+
+type FinalMessageProps = {
+    id: string,
+    to: string,
+    subject: string,
+    content: string,
+    parentCallback: Function,
+};
+
+export default function FinalMessage
+({ id, to, subject, content, parentCallback }: FinalMessageProps) {
+  const toMessage = to
+  const subjectMessage = subject
+  const idMail = id
+
+  const [isEditing, setIsEditing] = useState(false)
+  const [messageContent, setMessageContent] = useState(content)
+
+  const handleEditButton = () => {
+    setIsEditing(true)
+  }
+
+  const handleEditMessage = (e: any) => {
+    setMessageContent(e.target.value)
+  }
+
+  const handleSubmitButton = () => {
+    setIsEditing(false)
+    console.log(idMail)
+    parentCallback(idMail, to, subject, messageContent)
+  }
+
+  return (
+        <div>
+            <Typography variant="body1">To:{toMessage}</Typography>
```

Comment:
Let's add a space after To: so it becomes `To: {toMessage}`

to be consistent with the space after subject